daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Engine.scala

```diff
@@ -244,14 +245,16 @@ class Engine(val config: EngineConfig = new EngineConfig(LanguageVersion.StableV
       try {
         run
       } catch {
-        // The two following error should be prevented by the type checking does by translateCommand
+        // The following error should be prevented by the type checking does by translateCommand
```

Comment:
I think you wanted to place this above `PackageNotFound`. `SErrorCrash` is always internal.